src/ProductDefects.php

```diff
@@ -0,0 +1,239 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\ShopStandards\UsedGoods.
+ */
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * Used or defective goods user consent checkbox functionality.
+ */
+class ProductDefects {
+
+  /**
+   * Checkbox consent initialization method.
+   */
+  public static function init() {
+    // Logs consent to defective or used product when product is added to cart.
+    add_action('woocommerce_add_to_cart', function () {
+      ProductDefects::logConsent();
+    });
+
+    add_action('wp', function () {
```

Comment:
why is this an anonymous function and not a static method, like the rest?

Reply:
Type. Thanks!